bofire/data_models/surrogates/single_task_gp.py

```diff
@@ -99,16 +101,13 @@ class SingleTaskGPSurrogate(TrainableBotorchSurrogate):
     type: Literal["SingleTaskGPSurrogate"] = "SingleTaskGPSurrogate"
 
     kernel: AnyKernel = Field(
-        default_factory=lambda: ScaleKernel(
-            base_kernel=MaternKernel(
-                ard=True,
-                nu=2.5,
-                lengthscale_prior=BOTORCH_LENGTHCALE_PRIOR(),
-            ),
-            outputscale_prior=BOTORCH_SCALE_PRIOR(),
+        default_factory=lambda: MaternKernel(
```

Comment:
In the original paper and I think also in botorch they are using an RBF Kernel, or?